.jenkins/precheckin-cuda.groovy

```diff
@@ -89,6 +89,6 @@ ci: {
             properties(auxiliary.addCommonProperties(property))
     }
 
-    String hostBuildCommand = './install.sh -c --compiler=g++ --cuda'
+    String hostBuildCommand = 'HIP_PLATFORM=nvidia ./install.sh -c --compiler=g++'
```

Comment:
Okay so this is wrong @daineAMD.  Nothing in install.sh reads that env variable to set build configuration.  
I would suggest this needs to be added in separate PR.  Also the move from install.sh code duplication of rmake.py should be addressed soon.  I was trying to explain whether CI failures which seem to be using wrong reference library were due to CI pipeline bug or stale cache but this should be reverted first to see if it passes.  Then I can make fix PR or you can, we can discuss.  Same goes for moving build code from install into rmake I can look if you want.

Reply:
Yep I'll remove that now. I do have a ticket for fixing install.sh/rmake.py duplication, may or may not get to it this release.